ArduPlane/Attitude.cpp

```diff
@@ -840,13 +844,16 @@ void Plane::set_servos(void)
 			if (RC_Channel_aux::function_assigned(RC_Channel_aux::k_dspoiler1) && RC_Channel_aux::function_assigned(RC_Channel_aux::k_dspoiler2)) {
 				float ch3 = ch1;
 				float ch4 = ch2;
-				if ( BOOL_TO_SIGN(g.reverse_elevons) * channel_rudder->servo_out < 0) {
-				    ch1 += abs(channel_rudder->servo_out);
-				    ch3 -= abs(channel_rudder->servo_out);
-				} else {
-					ch2 += abs(channel_rudder->servo_out);
-				    ch4 -= abs(channel_rudder->servo_out);
-				}
+                   //get rudder value and multiply by DSPOILR_RUD_RATE/100:
```

Comment:
This code block doesn't need to be here as its only called for the old ELEVON_MIXING configuration which we don't really want to change as its legacy code.